README.md

```diff
@@ -27,14 +27,9 @@ php artisan vendor:publish --provider="AndreasElia\PostmanGenerator\PostmanGener
 
 ## Configuration
 
-You can modify the `api-postman.php` config values:
-
-- `structured` - If you want folders to be generated based on route names.
-- `base_url` - The base URL for all of your endpoints.
-- `auth_middleware` - The middleware which wraps your authenticated API routes.
-- `headers` - The headers applied to all routes within the collection.
-- `enable_formdata` - Determines whether or not form data should be handled.
-- `formdata` - The key/values to requests for form data dummy information.
```

Comment:
Can you drop this commit from the PR? Thanks.

Reply:
Hmm, no idea way this got committed. Will look into it.

Reply:
@tomirons I don't see how I could drop it - it was on the main branch when I forked and modified. I'll be happy to do it if you let me know how.